app/rpc/rpchandlers/get_mempool_entries.go

```diff
@@ -7,7 +7,45 @@ import (
 )
 
 // HandleGetMempoolEntries handles the respectively named RPC command
-func HandleGetMempoolEntries(context *rpccontext.Context, _ *router.Router, _ appmessage.Message) (appmessage.Message, error) {
+func HandleGetMempoolEntries(context *rpccontext.Context, _ *router.Router, request appmessage.Message) (appmessage.Message, error) {
+	getMempoolEntriesRequest := request.(*appmessage.GetMempoolEntriesRequestMessage)
+
+	var entries []*appmessage.MempoolEntry
+	var err error
+
+	if getMempoolEntriesRequest.IncludeTransactionPool && getMempoolEntriesRequest.IncludeOrphanPool { //both true
```

Comment:
You can convert it to only two `if`s. It could be written as something like this:
```
var entries []*appmessage.MempoolEntry

if getMempoolEntriesRequest.IncludeTransactionPool{
  ...
  entries = append(entries, transactionPoolEntries)
}

if getMempoolEntriesRequest.IncludeOrphanPool{
  ...
  entries = append(entries, orphanPoolEntries)
}
```

Reply:
done